backend/app/use_case/recommendation.py

```diff
@@ -129,14 +141,14 @@ def generate_recommendation_structured_format_query(
         )
 
     except Exception as e:
-        print("OpenAI error: ", e)
+        logger.error(f"OpenAI error: {e}")
         raise Exception("OpenAI service error")
 
     openai_functions_called = openai_response["choices"][0]["message"].get(
         "function_call"
     )
 
-    # print("openai_response", openai_response) # DEBUG
+    logger.debug(f"openai_response {openai_response}")  # DEBUG
```

Comment:
200 gpt response is not appear in the log
<img width="749" alt="image" src="https://github.com/chanon-mike/smart-ryokou/assets/27944646/584f5903-c3db-4c2a-b41c-9d839c67d753">